storage/cuprate-blockchain/src/service/read.rs

```diff
@@ -504,6 +504,12 @@ fn key_image_spent(env: &ConcreteEnv, key_images: HashSet<KeyImage>) -> Response
         key_image_exists(&key_image, tables.key_images())
     };
 
+    // FIXME:
+    // Create/use `enum cuprate_types::Exist { Does, DoesNot }`
+    // or similar instead of `bool` for clarity.
+    // <https://github.com/Cuprate/cuprate/pull/113#discussion_r1581536526>
+    //
+
     // Collect results using `rayon`.
```

Comment:
```suggestion
    // FIXME:
    // Create/use `enum cuprate_types::Exist { Does, DoesNot }`
    // or similar instead of `bool` for clarity.
    // <https://github.com/Cuprate/cuprate/pull/113#discussion_r1581536526>
    //
    // Collect results using `rayon`.
```